ext/curl/tests/bug45161.phpt

```diff
@@ -2,23 +2,25 @@
 Bug #45161 (Reusing a curl handle leaks memory)
 --SKIPIF--
 <?php
-if (substr(PHP_OS, 0, 3) == 'WIN') {
-	exit("skip not for Windows");
-}
-if (!extension_loaded("curl")) {
-	exit("skip curl extension not loaded");
-}
+include 'skipif.inc';
 $curl_version = curl_version();
 if ($curl_version['version_number'] < 0x071100) {
 	exit("skip: test works only with curl >= 7.17.0");
 }
+$ch = curl_init('http://127.0.0.1:9/');
+curl_exec($ch);
```

Comment:
Should the timeout be defined so this will not take too long?

Reply:
This code should actually have been removed. Thanks for noticing!